### ISSUE TITLE
Key Lights cycle off and on a few times per day

### ISSUE THREAD

**Issue by AngryMooses** [derjayjay/homebridge-keylights#426]
I can't tell where exactly the issue is coming from, but for the last month or so my lights will turn off and then back on again.  Often happens when I'm in the middle of a zoom meeting so I can't do much troubleshooting.  Nothing in the Homebridge logs that would point to an issue.

Anyone else having this issue, to help with my troubleshooting?

**Comment by derjayjay** [comment]
Hi @AngryMooses,

does the issue still exist? Have you checked for any open firmware updates in the Elgato control app?

**Comment by AngryMooses** [comment]
Yes, this is still happening.  I updated firmware last night again to verify, and just had the same behaviour occu5.  They tend to flicker on and off one after the other, I wonder if it could be related to polling of some sort?

**Comment by derjayjay** [comment]
Sorry to hear. I'm planning to do some improvements on the plugin over the summer and redo the polling system. Let's see if that will improve things for you.
